src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -151,6 +151,32 @@ public void ClusterOnDatabaseChanged(object sender, (string dbName, long index,
             }
         }
 
+        public void ClusterOnDatabaseValueChanged(object sender, (string dbName, long index, string type) t)
+        {
+            if (DatabasesCache.TryGetValue(t.dbName, out var task) == false)
+            {
+                // if the database isn't loaded, but it is relevant for this node, we need to create
```

Comment:
Sorry, I forgot to check against the db record (`record.Topology.RelevantFor(_serverStore.NodeTag)`